docs/docs/features/monitoring.md

```diff
@@ -0,0 +1,113 @@
+# Monitoring
+
+## Overview
+
+Immich provides a variety of performance metrics to allow for local monitoring and insights. This integration is primarily in the form of Prometheus metrics. However, exporting traces is also possible due to the use of OpenTelemetry instrumentation.
+
+:::note
+This is an opt-in feature intended for you to monitor immich's performance. This data isn't aggregated anywhere beyond what you've configured.
```

Comment:
```suggestion
This is an opt-in feature intended for you to monitor immich's performance. This data isn't sent anywhere beyond what you've configured.
```

I think this would be just a touch clearer (since it is potentially a sensitive subject :P)